pgmpy/models/LinearGaussianBayesianNetwork.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import division
 
+import logging
```

Comment:
@sandeep-n Thanks a lot for the PR. But could you please remove the blank line above this or even better move `import logging` above `from __future__ .. `. The imports in the codebase are done in the following order (with an empty line between them):
1. Python core imports.
2. Third-party package imports
3. Internal imports

Reply:
@ankurankan I removed the blank line. Trying to move it above the `from __future__` causes a `SyntaxError: from __future__ imports must occur at the beginning of the file`.

Reply:
Sorry, my bad. Forgot that future imports need to be at the top.